pkg/controller/operators/olm/operator.go

```diff
@@ -920,6 +959,29 @@ func (a *Operator) handleClusterServiceVersionDeletion(obj interface{}) {
 		syncError := a.objGCQueueSet.RequeueEvent("", kubestate.NewResourceEvent(kubestate.ResourceUpdated, cr))
 		logger.Debugf("handleCSVdeletion - requeued update event for %v, res=%v", cr, syncError)
 	}
+
+	webhookSelector := labels.SelectorFromSet(ownerutil.OwnerLabel(clusterServiceVersion, "ClusterServiceVersion")).String()
+	mWebhooks, err := a.opClient.KubernetesInterface().AdmissionregistrationV1().MutatingWebhookConfigurations().List(context.TODO(), metav1.ListOptions{LabelSelector: webhookSelector})
+	if err != nil {
+		logger.WithError(err).Warn("cannot list MutatingWebhookConfigurations")
+	}
+	logger.Infof("Alex: Deleteing %d mutatingWebhooks", len(mWebhooks.Items))
```

Comment:
nit: Not sure why your name is on the log? Debugging?

Reply:
leftovers! lol